features/dd-sdk-android-session-replay-compose/src/test/kotlin/com/datadog/android/sessionreplay/compose/internal/utils/ColorUtilsTest.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.compose.internal.utils
+
+import com.datadog.android.sessionreplay.compose.internal.utils.ColorUtils.Companion.RGBA_LENGTH
+import com.datadog.android.sessionreplay.compose.test.elmyr.SessionReplayComposeForgeConfigurator
+import fr.xgouchet.elmyr.Case
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.junit5.ForgeConfiguration
+import fr.xgouchet.elmyr.junit5.ForgeExtension
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.junit.jupiter.api.extension.Extensions
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.junit.jupiter.MockitoSettings
+import org.mockito.quality.Strictness
+
+@Extensions(
+    ExtendWith(MockitoExtension::class),
+    ExtendWith(ForgeExtension::class)
+)
+@MockitoSettings(strictness = Strictness.LENIENT)
+@ForgeConfiguration(SessionReplayComposeForgeConfigurator::class)
+internal class ColorUtilsTest {
+
+    private val testedUtils = ColorUtils()
+
+    @Test
+    fun `M return input value W convertRgbaToArgb() { length is less than 8 }`(
+        forge: Forge
+    ) {
+        // Given
+        val input = forge.anAsciiString(1)
+
+        // Then
+        assertThat(testedUtils.convertRgbaToArgb(input)).isEqualTo(input)
+    }
```

Comment:
so if we pass `rgb` string, the output will be wrong? Shouldn't we check in the `convertRgbaToArgb` method that input is **exactly** 8 characters (or 6 if we want to support `rgb`?

Otherwise we may have this function trying to compose something for the strings of format we don't expect.